katalog/configs/systemd/kubelet/systemd-index-template.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I wonder if it would be better to have the index pattern created from a `common` folder or in the parent folder `kustomization.yaml` file, otherwise if you don't deploy the kubelet config for some reason but you deploy the docker one for example it won't work.
Unless you have to always deploy the `kubelet` one.

Reply:
Since i merged all the common systemd into one hosttailer, I already did that 👍